DSIW/overtimer
# Add analysis modules

- Add Workday
- Add validation to Workday
- Add fromTimelogs
- Add statistics for min and max time
- Add median
- Rename
- Extract getTimeStatistics
- Extract TimeStatistic
- Extract formatTime
- Add tests
- Extract TimeLogTestFactory
- Fix eslint
- Add time statistics for end time
- Fix null check
- Remove getFormattedStartTime
- Add getPauseMs
- Extract Duration
- Add pause duration
- Fix eslint
- Fix compilation error
